mod/activity/views/default/resources/river.php

```diff
@@ -38,7 +38,7 @@
 		$subject = get_user_by_username($subject_username);
 		if (!$subject) {
 			register_error(elgg_echo('river:subject:invalid_subject'));
-			forward('');
+			forward();
```

Comment:
Does this change home page to REFERER?

Reply:
no... the forward function has `''` as the default, so it was pointless to provide it.
forward defaults to home... elgg_ok_response defaults to REFERER